hibernate-core/src/test/java/org/hibernate/test/flush/NonTransactionalDataAccessTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.test.flush;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.TransactionRequiredException;
+import java.io.Serializable;
+
+import org.hibernate.Session;
+import org.hibernate.cfg.AvailableSettings;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.resource.transaction.spi.TransactionStatus;
+
+import org.junit.Test;
+
+import org.hibernate.testing.TestForIssue;
+import org.hibernate.testing.junit4.BaseCoreFunctionalTestCase;
+
+import static org.hamcrest.core.IsNot.not;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Andrea Boriero
+ */
+@TestForIssue(jiraKey = "HHH-10877")
+public class NonTransactionalDataAccessTest extends BaseCoreFunctionalTestCase {
+
+	private String allowUpdateOperationOutsideTransaction = "true";
```

Comment:
It's the value for the second parameter of `configuration.setProperty(String,String); `
and it's not a constant, I want to change this inside the test 
```
allowUpdateOperationOutsideTransaction = "false";
rebuildSessionFactory();
```